### PR TITLE
[Admin] Get latest changes

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excel/excel.range.yml [hunk 1]
```diff
@@ -1328,8 +1328,11 @@ methods:
       Finds the given string based on the criteria specified. If the current
       range is larger than a single cell, then the search will be limited to
       that range, else the search will cover the entire sheet starting after
-      that cell. If there are no matches, this function will return a null
-      object.
+      that cell. If there are no matches, then this function will return an
+      object with its `isNullObject` property set to `true`<!-- -->. For further
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excel/excel.range.yml:1332]: The isNullObject comments all have `<!-- -->` included. Should we be concerned about this?

File: docs/docs-ref-autogen/excel/excel/excel.range.yml [hunk 2]
```diff
@@ -1879,10 +1882,10 @@ methods:
     package: excel!
     fullName: getImage()
     summary: >-
-      Renders the range as a base64-encoded png image. **Important**: This API
-      is currently unsupported in Excel for Mac. Visit [OfficeDev/office-js
-      Issue \#235](https://github.com/OfficeDev/office-js/issues/235) for the
-      current status.
+      Renders the range as a base64-encoded png image. *Important**: This API is
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excel/excel.range.yml:1885]:
```suggestion
      Renders the range as a base64-encoded png image. **Important**: This API is
```

File: docs/docs-ref-autogen/excel/excel/excel.range.yml [hunk 3]
```diff
@@ -1879,10 +1882,10 @@ methods:
...
+      currently unsupported in Excel for Mac. Visit \[OfficeDev/office-js Issue
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excel/excel.range.yml:1886]: This change seems to have added extra backslashes ( \ ) before the link and at the end of the link on the next line. 
```suggestion
      currently unsupported in Excel for Mac. Visit [OfficeDev/office-js Issue
```

**Review by alison-mk — APPROVED** [review]
Approved Excel changes, with a couple questions.

File: docs/docs-ref-autogen/excel/excel/excel.range.yml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/excel/excel/excel.range.yml:1885]: @alison-mk @Rick-Kirkham You'll need to update this in DefinitelyTyped

File: docs/docs-ref-autogen/excel/excel/excel.range.yml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/excel/excel/excel.range.yml:1332]: @alison-mk For things like this, you can verify the behavior by checking the review site for this branch. Let me know if you need the link

File: docs/docs-ref-autogen/excel/excel/excel.range.yml (hunk 3 above)
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/excel/excel/excel.range.yml:1886]: @alison-mk Same here - you can verify the behavior by checking the review site for this branch

File: docs/docs-ref-autogen/excel/excel/excel.eventtype.yml [hunk 4]
```diff
@@ -239,7 +246,7 @@ fields:
     package: excel!
     summary: >-
       WorksheetFormulaChanged represents the type of event registered on a
-      worksheet, and occurs when a formula is changed.
```
Comment (Rick-Kirkham) [inline docs/docs-ref-autogen/excel/excel/excel.eventtype.yml:242]: This change should be reverted.

**Review by Rick-Kirkham — APPROVED** [review]

File: docs/docs-ref-autogen/excel/excel/excel.eventtype.yml (hunk 4 above)
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/excel/excel/excel.eventtype.yml:242]: @Rick-Kirkham Can you make sure this is fixed in DefinitelyTyped? Thx

File: docs/docs-ref-autogen/excel/excel/excel.range.yml (hunk 1 above)
Comment (alison-mk) [reply docs/docs-ref-autogen/excel/excel/excel.range.yml:1332]: Thanks! The review site output looks fine to me.

**Pull request merged**